### PR TITLE
Add docs for Bookmarks plugin

### LINKED ISSUE #454: Document Bookmarks core plugin
At least 1 user has already bumped against the absence of Bookmarks docs. <https://discord.com/channels/686053708261228577/694233507500916796/1103444555006759063>

### LINKED ISSUE #455: Remove or deprecate Starred core plugin docs
Since the Starred core plugin has been removed, either remove the docs or update them to note that it's been removed from the current version and replaced by Bookmarks.

### PR DESCRIPTION
**Pull request by marcusolsson**
Deprecates docs for the Starred plugin in favor of the Bookmarks plugin.

Fixes #455 
Fixes #454

### CODE REVIEW

File: en/Plugins/Bookmarks.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Or, open the Bookmarks tab by running **Bookmarks: Show bookmarks** from the [[Command palette]].
+
+Open a bookmark by selecting it in the Bookmarks tab.
+
+You can reorder bookmarks by selecting and dragging them.
+
+## Add a bookmark
+
+To bookma rk a note:
```
Comment (liamcain) [inline en/Plugins/Bookmarks.md:17]:
```suggestion
To bookmark a note:
```

File: en/Plugins/Bookmarks.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+alias: Plugins/Starred
+---
+
+You can bookmark files and organize them into bookmark groups to find them easier.
```
Comment (liamcain) [inline en/Plugins/Bookmarks.md:5]: We should mention that you can bookmark more than just files.

Comment (liamcain) [reply en/Plugins/Bookmarks.md:5]: I think there's so copy you can borrow from the Bookmarks announcement message.

File: en/Plugins/Bookmarks.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+- Select **Bookmark the active tab** (bookmark with plus icon) at the top of the Bookmarks tab.
+
+**File explorer:**
+
+- Right-click a file in [[File explorer]] and click **Bookmark**.
+
+**Command palette**:
+
+1. Press `Ctrl+P` (or `Cmd+P` on macOS) to open the [[Command palette]].
+2. Search for the **Bookmark** command.
```
Comment (liamcain) [inline en/Plugins/Bookmarks.md:30]:
```suggestion
2. Search for the **Bookmark...** command.
```

File: en/Plugins/Bookmarks.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+3. Press `Enter` with the command selected to toggle the active note.
```
Comment (liamcain) [inline en/Plugins/Bookmarks.md:31]: This doesn't toggle the active note. This opens the bookmark modal which gives you the option to add a title, select a bookmark group or (if the bookmark already exists) the option to remove the bookmark.

File: en/Plugins/Bookmarks.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+## Remove a bookmark
+
+In the Bookmarks tab, right-click the bookmark and select **Remove**.
```
Comment (liamcain) [inline en/Plugins/Bookmarks.md:35]: You can also remove a bookmark from the bookmark modal.

File: en/Plugins/Bookmarks.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+To create a bookmark group:
+
+**Bookmarks tab**:
+
+- Select **New bookmark group** (folder with plus icon) at the top of the Bookmarks tab.
+
+**Options menu:**
+
+- Click **More options** in the upper-right corner of a note, and then click **Bookmark X tabs**.
```
Comment (liamcain) [inline en/Plugins/Bookmarks.md:47]: At least to me, this seems like it belongs in it's own heading "Bookmark multiple items"

If we add another heading for that, we can also mention you can multiselect items in the file explorer and right-click, select "bookmark..."

File: en/Plugins/Bookmarks.md (hunk 3 above)
Comment (marcusolsson) [reply en/Plugins/Bookmarks.md:30]: This is following the Microsoft Style Guide. Do you feel strongly enough to add an exception for this?

> When there's an ellipsis in UI, don’t include it in instructions or procedures.

https://learn.microsoft.com/en-us/style-guide/punctuation/ellipses

File: en/Plugins/Bookmarks.md (hunk 6 above)
Comment (marcusolsson) [reply en/Plugins/Bookmarks.md:47]: Oh! Didn't realize you could do that, that seems to justify another heading 👍

File: en/Plugins/Bookmarks.md (hunk 3 above)
Comment (liamcain) [reply en/Plugins/Bookmarks.md:30]: oh okay. that's fine then

**Pull request merged**